lib/EditableList/readme.md

```diff
@@ -31,6 +31,7 @@ Use the `EditableList` component in your jsx
 ### Configuration (props)
 Name | type | description | default | required
 --- | --- | --- | --- | ---
+canCreate | boolean | Determines if the '+ New' button is active. | | no
```

Comment:
What about setting the "default" column to `false`?

Reply:
I'm pretty sure it's undefined if not present so only falsy not false and that column reflects that by being empty for many optional properties.

Reply:
...hm, but actually it's true now that I see it in the table. Another PR I guess....

Reply:
Leave it how it is — your previous argument was convincing.

Reply:
Except that I was confused while making that previous argument and it turns out to be explicitly defaulted to true so it was most honest to be explicit about that default and indeed consistent with another column further down documented as true. So I'd gone ahead with another PR to add that.

Reply:
Yep, saw it, it's good.